env/git.go

```diff
@@ -120,9 +121,11 @@ var GitBlob = func(path string, commit *object.Commit) (string, error) {
 		return blob, nil
 	}
 
+	logrus.Debug("Getting git blob_id for source file %s", path)
 	file, err := commit.File(path)
 
 	if err != nil {
+		logrus.Debug("failed to find file %s", path)
```

Comment:
Do we have a consistent style for starting these log messages with a capital or lowercase letter?